test/services/database-analyzer/database-analyzer-sequelize.js

```diff
@@ -35,6 +36,31 @@ describe('Database analyser > Sequelize', () => {
         await sequelizeHelper.close();
       });
 
+      it('should give us constraint_name, table_name_ column_type, column_name, foreign_table_name, foreign_column_name, unique_indexes', async () => {
+        const tableForeignKeysAnalyzer = new TableForeignKeysAnalyzer(databaseConnection, 'public');
+        const constraints = await tableForeignKeysAnalyzer.perform('addresses');
+
+        if (dialect === 'postgres') {
```

Comment:
We are in a ['postgres', 'MySQL'] loop. It's tricky to have this kind of `if..else..` here.
Please remove the if, replacing it by a map construction.
```
const actualInfos = Object.keys(constraints[0]).sort();
expect(actualInfos).to.eql(expectedInfos[dialect])
```
I let you write the expectedInfos map.

Reply:
See my comment above, the if else will disappear once we add the other scripts.